### ISSUE TITLE
Task always started with newest revision of app

### ISSUE THREAD

**Issue by bjoernpollex** [sbg/sevenbridges-python#42]
When I fetch a specific older revision of an app, and then start a task using that revision, the task will be spawned with the newest revision of that app:

```
>>> app = api.apps.get_revision(id='whatever', revision=0)
>>> task = api.tasks.create('name', project='some-project', app, inputs={}, run=True)
>>> task.app
'whatever/3'
```

I'm running version 0.5.3 on Python 3.5.

**Comment by SenadI** [comment]
@bjoernpollex Thanks for noticing this one it is a reintroduced bug due to changes in APP id's that we did in older version. I will try to fix it soon. It may require more complex looking to solve this one.

**Comment by SenadI** [comment]
@bjoernpollex Fixed it in v0.5.4.
